src/store/modules/data/index.js

```diff
@@ -1,41 +1,59 @@
 import configRepo from '~/repo/configRepo'
-import { VALID_PLATFORMS } from '~/lib/constants'
+import { flattenLayers, getLayersTags } from '~/lib/layer-helpers'
 
 export default {
   namespaced: true,
 
   state: () => ({
-    layers: [],
+    // originalLayers: [],
```

Comment:
I keep this around just until we're sure we don't need it. Check definitively after we have rendered the layers on the map, and figured out what happens if you switch between 'all layers' and 'filtered flattened layers' by using the multiselect tag selector